_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+title:  "Building your own Kafka Connect image with a custom resource"
+date: 2021-03-23
+author: jakub_stejskal
+---
+In a [previous blog post](https://strimzi.io/blog/2020/05/07/camel-kafka-connectors/) we showed how easy it is to integrate [Camel Kafka Connectors](https://camel.apache.org/camel-kafka-connector/latest/) with Strimzi by configuring a `KafkaConnect` custom resource.
+That approach had one limitation - you had to build your own Kafka Connect image and use it in the custom resource.
+This step is no longer needed thanks to a feature introduced in Strimzi 0.22 that allows custom Kafka Connect images to be built directly by Strimzi.
+
+### How does it work?
+
+Strimzi builds the image in two ways based on the underlying cluster.
+If you use Kubernetes, the image is built by [kaniko](https://github.com/GoogleContainerTools/kaniko).
+kaniko is a tool to build container images from a Dockerfile, inside a container or Kubernetes cluster.
+kaniko doesn't depend on a Docker daemon and executes each command within a Dockerfile completely in userspace.
+This enables building container images in environments that can't easily or securely run a Docker daemon, such as a standard Kubernetes cluster.
+If you use Openshift, the image is built by Openshift Builds.
+
+In both cases, Strimzi will spin-up a build pod, which builds the image based on the configuration from the custom resource.
+The final image is then pushed into a specific container registry or image stream, again based on the configuration. 
+The Kafka Connect cluster specified by the custom resource with the build configuration part will then use the newly built image.
+
+### Kafka Connect configuration
+
+A new `build` configuration for the `KafkaConnect` resource allows you to configure a list of custom connectors, which are downloaded and _baked into_ a new `KafkaConnect` image specified by you.
+
+```yaml
```

Comment:
Maybe a title for the yaml? .Kafka Connect configuration to create a new image automatically